ddl/ddl_db_change_test.go

```diff
@@ -292,14 +292,38 @@ type sqlWithErr struct {
 	expectErr error
 }
 
+type expectQuery struct {
+	sql  string
+	rows []string
+}
+
+// https://github.com/pingcap/tidb/pull/6249 fix the following two test cases.
```

Comment:
fixes?

Reply:
If the no #6294 , the following two cases should fail.

Reply:
is it a spelling mistake?

Reply:
Fixed.